src/app/api/ai/route.ts

```
@@ -0,0 +1,102 @@
+    }
+
+    switch (validatedAIOperations.data) {
+        case "search":
+            const result = streamText({
+                model: aiProvider(validatedAI.data.model),
+                messages,
+            });
+
+            return result.toDataStreamResponse();
+        case "mock":
+            const validatedMockPayload = MockPayloadSchema.safeParse(mock);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)